distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -199,37 +198,64 @@ When searching with wildcards, do not include the punctuation at the beginning o
 Text searches are by default case insensitive, but case sensitive searches are an option.
 
 [[_temporal_search_details]]Temporal Search Details:: Search based on absolute time of the created, modified, or effective date.
```

Comment:
The indentation of the temporal Search details section is not consistent with `Text Search Details`, `Spatial Search Details`, and `Type Search Details`.
![image](https://user-images.githubusercontent.com/8041246/51354747-df361180-1a71-11e9-9a5f-0c3a66ac06e9.png)